### ISSUE TITLE
super-tiled2unity.v2.1.0 Collider2DExtensions exceptions using Unity2022.3.9

### ISSUE THREAD

**Issue by streamcypher** [Seanba/SuperTiled2Unity#254]
'Collider2D' does not contain a definition for 'compositeCapable' and no accessible extension method 'compositeCapable' accepting a first argument of type 'Collider2D' could be found (are you missing a using directive or an assembly reference?)

 'Collider2D' does not contain a definition for 'compositeOperation' and no accessible extension method 'compositeOperation' accepting a first argument of type 'Collider2D' could be found (are you missing a using directive or an assembly reference?)

For Unity2022.3.9, Collider2d cannot find compoisteCapable and compositeOperation properties.

if (merge && !collider2d.compositeCapable)
{
    Debug.LogWarning($"Super Tiled2Unity warning: Collider2D '{collider2d.name}' is not composite capable");
}
collider2d.compositeOperation = merge ? Collider2D.CompositeOperation.Merge : Collider2D.CompositeOperation.None;

**Comment by c0d3r9** [comment]
I look at the docs at it seems it not available in 2022.3 but available in 2023.1 and above:
Maybe it should be a switch inside the code for different unity versions.
For me i havent´t tested 2.1.0 atm but i use 2023.2 because my game grow with this version. ;)

**Comment by streamcypher** [comment]
> I look at the docs at it seems it not available in 2022.3 but available in 2023.1 and above: Maybe it should be a switch inside the code for different unity versions. For me i havent´t tested 2.1.0 atm but i use 2023.2 because my game grow with this version. ;)

Yeah, I change UNITY_2022_3_OR_NEWER to UNITY_2023_3_OR_NEWER, the error is cleared, but importing the sample Overhead tmx failed. #253

**Comment by Seanba** [comment]
Hi there, @streamcypher. Thanks for letting me know. Keeping track of all these Unity versions is pain.
This is fixed now with [version 2.1.1](https://github.com/Seanba/SuperTiled2Unity/blob/master/deploy/super-tiled2unity.v2.1.1.zip)

I'll address your issue with the Overhead tmx in #253.
